apps/antalmanac/src/providers/Themev5.tsx

```diff
@@ -37,32 +36,27 @@ interface Props {
  * sets and provides the MUI theme for the app
  */
 export default function AppThemev5Provider(props: Props) {
-    const [darkMode, setDarkMode] = useState(isDarkMode());
+    const { theme, setTheme } = useThemeStore();
 
     useEffect(() => {
-        AppStore.on('themeToggle', () => {
-            setDarkMode(isDarkMode());
-        });
         window.matchMedia('(prefers-color-scheme: dark)').addEventListener('change', (e) => {
-            if (AppStore.getTheme() === 'auto') {
-                setDarkMode(e.matches);
-            }
+            setTheme(e.matches ? 'dark' : 'light');
```

Comment:
Same advice as the other about removing the event handlers when the component unmounts.